packages/forms-web-app/src/dynamic-forms/s78-questionnaire/journey.js

```diff
@@ -48,8 +48,10 @@ class S78Journey extends Journey {
 			new Section('Consultation responses and representations', 'consultation'),
 			new Section('Planning officer’s report and relevant policies', 'planning-officer-report')
 				.addQuestion(questions.emergingPlan)
-				.addQuestion(questions.communityInfrastructureLevyAdopted),
-			new Section('Site access', 'site-access').addQuestion(questions.neighbouringSite),
+				.addQuestion(questions.communityInfrastructureLevy),
```

Comment:
does this need to stay communityInfrastructureLevyAdopted?

Reply:
(community infrastructure levy stuff also needs to be in Planning officer's section)

Reply:
Good spot didn't notice I had changed that